core/templates/dev/head/components/profileLink.js

```diff
@@ -0,0 +1,21 @@
+oppia.directive('profileLinkText', [function() {
+  return {
+    restrict: 'E',
+    scope: {
+      linkedName: '@'
```

Comment:
How about calling this 'username', here and in the other directive? I think it makes it clearer what the data represents (as opposed to what we subsequently try to do with it).